SPECS/perl-Test-Pod-Coverage/perl-Test-Pod-Coverage.spec

```diff
@@ -1,29 +1,36 @@
+Summary:        Check for pod coverage in your distribution
 Name:           perl-Test-Pod-Coverage
 Version:        1.10
-Release:        17%{?dist}
-Summary:        Check for pod coverage in your distribution
+Release:        18%{?dist}
 License:        Artistic 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://metacpan.org/release/Test-Pod-Coverage
 Source0:        https://cpan.metacpan.org/authors/id/N/NE/NEILB/Test-Pod-Coverage-%{version}.tar.gz#/perl-Test-Pod-Coverage-%{version}.tar.gz
+Source1:        LICENSE.PTR
```

Comment:
I think we have missed to include this file, which breaks the `srpm` packer.

Reply:
Ah, great catch, thanks!